contents/blog/the-posthog-array-1-33-0.md

```diff
@@ -0,0 +1,130 @@
+<b>IMPORTANT!</b> From this version onwards, you'll need to make sure to run all async migrations on your instance after upgrading. The next version (1.34.0) will not run until async migrations are completed. Please check out the <a href="/docs/self-host/configure/async-migrations/overview" target="_blank">async migrations</a> docs for details.
+</blockquote>
+
+## PostHog 1.33.0 release notes
+
+> To upgrade your PostHog instance, you can check out our [upgrade guide](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Experimentation](#new-experimentation)
+- [Dashboard permissions](#new-dashboard-permissions)
+- [Altinity Cloud support](#new-altitiny-cloud-support)
+- [Instance configuration UI](#new-instance-configuration-ui)
+
+### New: Experimentation
+
+PostHog now offers a full end-to-end A/B testing suite! Previously you could use [Feature Flags](/product/feature-flags) to run experiments, but it took a lot of manual work to setup and interpret. We wanted to fix that. This new experimentation suite will help you plan tests by selecting users, sample sizes and run times, then track results and let you know when results are statistically significant.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-experiments.png" alt="Screenshot: Experiments view" />
+
+With [Experimentation](/docs/user-guides/experimentation), you can now test multiple variants to optimize for a specific metric, or even conversion rate for a funnel. This enables you to ship product changes confidently while maintaining speed.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-experiments-2.png" alt="Screenshot: Experiments with a funnel conversion rate target metric" />
+
+> 🎁 Experimentation is a premium feature and requires a PostHog Scale or Enterprise license. [Learn more](/pricing).
+
+<br />
+
+
+### New: Dashboard permissions
+
+To introduce more detailed control to dashboards for large teams, PostHog now lets you select different edit permissions for each dashboard. You can select whether anyone can edit a dashboard or just a specific list of team members. This can help prevent accidental edits and provide confidence that the dashboards are showing the metrics you intend. Underlying insights in a dashboard are also protected from edit when this setting is enabled.
+
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-dashboard-permissions.png" alt="Screenshot: Dashboard permissions" />
+
+> 🎁 Dashboard permissions is a premium feature and requires a **PostHog Enterprise** license. [Learn more](/pricing).
+
+<br />
+
+### New: Altitiny Cloud support
+
+We've partnered up with Altinity to offer a worry-free managed ClickHouse database. Altinity is an expert in Clickhouse and data infrastructure and it can help large organizations to deploy, manage and scale their PostHog instance.
```

Comment:
```suggestion
We've partnered up with Altinity to offer a worry-free managed ClickHouse database. Altinity is an expert in Clickhouse and data infrastructure and it can help large organizations to deploy, manage and scale their PostHog instance. If you need to self-host PostHog but don't want to manage the deployment yourself, Altitiny is the perfect solution for you. 
```